server/models/Team.js

```diff
@@ -142,6 +142,7 @@ Team.prototype.provisionFirstCollection = async function (userId) {
     name: "Welcome",
     description:
       "This collection is a quick guide to what Outline is all about. Feel free to delete this collection once your team is up to speed with the basics!",
+    type: "atlas",
```

Comment:
Can remove this, guess you might have been on an old fork at one point?

Reply:
ups yes :D